packages/account/src/Sections/Security/ConnectedApps/connected-apps-article.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { localize, Localize } from '@deriv/translations';
+import AccountArticle from 'Components/article';
+
+const openAPIManagingWebsite = () => {
+    window.open(
+        'https://community.deriv.com/t/api-tokens-managing-access-on-third-party-applications-and-mobile-apps/29159',
+        '_blank'
+    );
+};
+
+const ConnectedAppsArticle = () => (
+    <AccountArticle
+        className='connected-apps__article'
+        title={localize('Learn more about APIs')}
+        descriptions={[
+            <Localize
+                key={0}
```

Comment:
Do we need this key property here?

Reply:
@amina-deriv Need to provide a key for the element if not it will give me error asking for missing key.
![Screenshot 2022-11-09 at 11 47 19 AM](https://user-images.githubusercontent.com/103026762/200733694-dd3fe607-bbb7-4e92-9aa6-31948ad8568c.png)